### PR TITLE
Updated installation steps

### PR DESCRIPTION
**Pull request by btkcodedev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,29 +26,29 @@ Technique of Encryption
   <h4>LSB based Image steganography:</h4>
```
Comment (BreadGenie) [inline README.md]: the heading here is not according to the hierarchy. Can you maybe change that?

Comment (BreadGenie) [reply README.md]: And the ``` ## Contributing ``` at line 99

Comment (btkcodedev) [reply README.md]: Done @BreadGenie

**Comment by BreadGenie** [comment]
Thanks for the contribution @btkcodedev!

**Pull request merged**